.github/workflows/ci.yml

```
@@ -0,0 +1,56 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.18
```

I would probably remove the Go version since that will be logged after anyways. Also useful so we don't have to update go version in too many places whenever we upgrade.

```suggestion
    - name: Set up Go
```